### ISSUE TITLE
version 1.7.2 x509: certificate has expired

### ISSUE THREAD

**Issue by Allenhaozi** [kubernetes/kubeadm#1119]
kubeadm version 1.7.2
kubernetes version 1.7.2

apiserver.crt x509: certificate has expired 

how to update master and node cerfificates

**Comment by danroliver** [comment]
I had this issue as well. Did you try the steps that I laid out here: https://github.com/kubernetes/kubeadm/issues/581#issuecomment-421477139

Where did it go wrong for you?

**Comment by neolit123** [comment]
hi,
1.7.x is outside of the supported version range and the kubeadm team doesn't have the bandwidth to support these old versions.

the tutorial that @danroliver posted here https://github.com/kubernetes/kubeadm/issues/581#issuecomment-421477139
should be sufficient to perform the rotation.

/close

please also feel free to continue your discussion in the closed issues.
thanks for understanding.

**Comment by guyplusplus** [comment]
On k8s 1.7 I faced a similar problem (x509 error included inside /var/log/kube-apiserver.log) and could not find any certificate expired. We decided to restart the apiserver docker on the master node. It resolved the problem.

$ sudo docker ps -a | grep apiserver
af99f816c7ec        gcr.io/google_containers/kube-apiserver@sha256:53b987e5a2932bdaff88497081b488e3b56af5b6a14891895b08703129477d85               "/bin/sh -c '/usr/loc"   15 months ago       Up 19 hours                                     k8s_kube-apiserver_kube-apiserver-ip-xxxxxc_0
40f3a18050c3        gcr.io/google_containers/pause-amd64:3.0                                                                                      "/pause"                 15 months ago       Up 15 months                                    k8s_POD_kube-apiserver-ip-xxxc_0
$ sudo docker restart af99f816c7ec
af99f816c7ec
$
